Simulation.cpp

```diff
@@ -49,11 +49,14 @@ Simulation::Simulation(QWidget *parent){
 
     // Initial Settings
     simulationStarted = false;
+    mm= ss= 0;
 
     drawGUI();
+    drawTimer();
```

Comment:
All draw calls should be inside the `drawGUI()` function. Everything else looks good, good job.